File: Labs2/Models/Students.cs
namespace Labs2.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    public partial class Students
    {
        [Key]
        public int ID_Student { get; set; }

        public string FIO { get; set; }

        [StringLength(10)]
        public string Gender { get; set; }

        [Column(TypeName = "date")]
        public DateTime? Date_Of_Birth { get; set; }

        public string Parents { get; set; }

        public string Address { get; set; }

        public string Phone { get; set; }

        public string Pasport_num { get; set; }

        public int? Credit_Card_Number { get; set; }

        [Column(TypeName = "date")]
        public DateTime? Date_Of_Receipt { get; set; }

        public string Group { get; set; }

        public int? Course { get; set; }

        public int? ID_Specialties { get; set; }

        public bool? Full_Time_Education { get; set; }

        public virtual Evaluations Evaluations { get; set; }

        public virtual Specialties Specialties { get; set; }
    }
}

File: lab2/Students.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity.Spatial

Partial Public Class Students
    <Key>
    Public Property ID_Student As Integer

    Public Property FIO As String

    <StringLength(10)>
    Public Property Gender As String

    <Column(TypeName:="date")>
    Public Property Date_Of_Birth As Date?

    Public Property Parents As String

    Public Property Address As String

    Public Property Phone As String

    Public Property Pasport_num As String

    Public Property Credit_Card_Number As Integer?

    <Column(TypeName:="date")>
    Public Property Date_Of_Receipt As Date?

    Public Property Group As String

    Public Property Course As Integer?

    Public Property ID_Specialties As Integer?

    Public Property Full_Time_Education As Boolean?

    Public Overridable Property Evaluations As Evaluations

    Public Overridable Property Specialties As Specialties
End Class
